### PR TITLE
Add information on sketchnoting and pricing

### PR DESCRIPTION
**Pull request by Malwine**
# One step at a time.
There is still a lot to do.
This time I added more information and shifted the containers around.
Thank you for reviewing.

```
What is Sketchnoting?

Sketchnotes are concise visual transcriptions of content.
They consist of notes and drawings. After their creation they get
posted to social media. High engagement rates make sketchnotes
a great tool to promote your content online.

What do I get?

There are different packages and advantages. These may include:
- creation of sketchnotes for your content
- high quality scans
- social media outreach and positive attention for your content

If you're interested the collaborator of your choice will send you 
a catalog of their services.


What is the price?

It depends on the packages you will book and if possible discounts apply. 
Think about what you invest in a professional photographer for your event 
and you are probably getting close. We chose the term "investing" with care btw.

Sketchnotes live long after your event is over. 
They are quick to grasp and simple to share. 
They make your content accessible and approachable. 
In short: what could be better promotion for your event? Or for your next event?

```

### CODE REVIEW

**Comment by daiyi** [comment]
I took your excellent points and tried to tell the story of Sketchnote Club and the beautiful dream we wish to bring into the world! Use as little or as much as you please

--
### What is Sketchnoting?

Sketchnotes are real-time transcriptions of live events (talks, lectures, or workshops), condensing complex information into digestible and visually interesting notes, diagrams, and drawings. They are ready to be distributed to social media right at the moment the event is over. High engagement rates make sketchnotes a great tool to promote your content and ideas online on the day of your event and beyond!

[insert examples here!]

### What is the Sketchnote Club?

The Sketchnote Club is a network of diverse folks who love teaching, learning, and storytelling at the intersection of tech, art, and other creative areas. Your content will be shared far and wide amongst us and to all of the communities we are a part of!

### What do I get?

There are different packages and advantages. These may include:
- sketchnotes at your event! (We love conferences and local meetups)
- high quality scans
- originals as gifts for speakers
- social media outreach and creative content marketing

If you are interested, reach out to the individuals of the Sketchnote Club you'd like collaborate with to discuss how you can work together at your next event. (add link to our contact info)

### What is the price?

Sketchnoting is hard work and a labour of love <3 We want to bring sketchnoting to our communities as an artistic and personal way of capturing events and ideas, we want this special way of sharing and teaching to enter the culture of technical meetups and conferences, and gain recognition like any other creative freelancing role (think: hiring a photographer for your wedding, or a live band for your holiday party).

Sketchnotes live long after your event is over. 
They are quick to grasp and simple to share. 
They make your content accessible, approachable, and add style and personality! 
They're a visually curious and creative way to spread awareness for your event.
